### PR TITLE
Waf disable enable

### PR DESCRIPTION
**Pull request by oscarDovao**
This Pr contains the functionality to disable and re-enable a Fastly service WAF. Disabling or enabling a service does not remove rules, change configuration nor create a new WAF version, it just mark the WAF as disable.

### CODE REVIEW

File: fastly/fixtures/wafs/deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      User-Agent:
+      - FastlyGo/1.3.0 (+github.com/fastly/go-fastly; go1.13.4)
+    url: https://api.fastly.com/waf/firewalls/5ui1qC2Y9TZIonO95zU3gM/versions/1
+    method: POST
+  response:
+    body: '{"errors":[],"data":null}'
+    headers:
+      Accept-Ranges:
+      - bytes
+      - bytes
```
Comment (trentrosenbaum) [inline fastly/fixtures/wafs/deploy.yaml]: I remember on previous PRs the duplicates were asked to be removed.  I am just raising here and I will only mark in this file, but applies to test data in the other yaml file.

File: fastly/fixtures/wafs/deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      - application/vnd.api+json
+      Date:
+      - Fri, 13 Dec 2019 08:52:53 GMT
+      Status:
+      - 202 Accepted
+      Strict-Transport-Security:
+      - max-age=31536000
+      Via:
+      - 1.1 varnish
+      - 1.1 varnish
```
Comment (trentrosenbaum) [inline fastly/fixtures/wafs/deploy.yaml]: duplicate entries

File: fastly/fixtures/wafs/deploy.yaml (hunk 1 above)
Comment (oscarDovao) [reply fastly/fixtures/wafs/deploy.yaml]: fixed

File: fastly/fixtures/wafs/deploy.yaml (hunk 2 above)
Comment (oscarDovao) [reply fastly/fixtures/wafs/deploy.yaml]: fixed

**Pull request closed without merging**